includes/admin/class-wcmp-export-consignments.php

```diff
@@ -552,7 +552,7 @@ private function setPhysicalProperties(): void
         $totalWeight = $this->getTotalWeight($orderWeight);
 
         if ($packageType === AbstractConsignment::PACKAGE_TYPE_DIGITAL_STAMP) {
-            $totalWeight = $extraOptions['weight'];
+            $totalWeight = $extraOptions['weight'] + (int) $this->getSetting(WCMYPA_Settings::SETTING_EMPTY_PARCEL_WEIGHT);
```

Comment:
plaats de `(int) $this->getSetting(WCMYPA_Settings::SETTING_EMPTY_PARCEL_WEIGHT)` even in een nieuwe variabelen, dan is de code overzichtelijker

Reply:
done